Makefile

```diff
@@ -17,34 +17,34 @@ quality: ## run quality checks using tox
 	tox -e quality
 
 piptools:
-	pip install -r pip-tools.txt
+	pip install -r requirements/pip-tools.txt
 
 _upgrade: piptools
-	# Generate pins for test requirements.
-	pip-compile test_requirements.in --upgrade -o test_requirements.txt
-
-	# Delete django pin from test_requirements.txt so that tox can control
+	pip-compile requirements/base.in --upgrade -o requirements/base.txt
+	pip-compile requirements/test.in --upgrade -o requirements/test.txt
+	pip-compile requirements/docs.in --upgrade -o requirements/docs.txt
```

Comment:
We should probably add the `--rebuild` option to all these pip-compile lines also.